### PR TITLE
move reco_utils/nni to reco_utils/tuning/nni

### PR DESCRIPTION
**Pull request by miguelgfierro**
### Description

move reco_utils/nni to reco_utils/tuning/nni to have only general concepts in reco_utils

### Related Issues

https://github.com/microsoft/recommenders/issues/689

### Checklist:

- [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by anargyri** [comment]
Since we do not have unit tests for the tuning notebooks, could you perform the integration tests before merging this PR?

**Comment by anargyri** [comment]
These are not the only changes needed, because svd_training.py has now also moved. The locations of this script inside the NNI notebook should also be modified.

**Comment by miguelgfierro** [comment]
mmm I know what you mean, let me take a deeper look

**Comment by anargyri** [comment]
Another note is that we have the tuning scripts for AzureML under the azureml folder. Should we move those under the tuning folder too or keep them where they are?

**Comment by miguelgfierro** [comment]
> Another note is that we have the tuning scripts for AzureML under the azureml folder. Should we move those under the tuning folder too or keep them where they are?

We can do it. Maybe we should do it in a different PR

**Comment by gramhagen** [comment]
there's also the evaluation/parameter_sweep module, which seems like it would fit better under tuning

File: reco_utils/tuning/nni/svd_training.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import nni
 import surprise
 import sys
-sys.path.append("../../")
+sys.path.append("../../../")
```
Comment (gramhagen) [inline reco_utils/tuning/nni/svd_training.py]: is this needed?

**Comment by yueguoguo** [comment]
LGTM

File: reco_utils/tuning/nni/svd_training.py (hunk 1 above)
Comment (miguelgfierro) [reply reco_utils/tuning/nni/svd_training.py]: yeah it is needed, it won't work otherwise

**Pull request merged**